proctor-consumer/src/main/java/com/indeed/proctor/consumer/AbstractGroups.java

```diff
@@ -454,6 +456,25 @@ protected final void appendTestGroupsWithAllocations(
         }
     }
 
+    private boolean filterRolledOutAllocations(final String testName) {
```

Comment:
nit: I'm terrible with names, but I'd prefer this to indicate the return type of `boolean`. Something like `isTestRolledOut`

Reply:
Thinking about this more. isTestRolledOut is not actually descriptive of what this method is doing. It is checking if the test should be logged when the Allocation is fully rolled out to 100%. A better name would be something like `shouldLogRolledOutTest`

Reply:
That makes sense. I tend to name methods more for their content than where they're used. So, I like the second option more since it indicates a `boolean` and what it is doing. Maybe even `isRolledOutAllocation`. Either way, I trust your judgement, so I'll leave it up to you.